api/proto/security_image/v1/security_image.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package ezex.users.security_image.v1;
+
+option go_package = "github.com/ezex-io/ezex-users/api/gen/go/security_image/v1;security_imagev1";
+
+// SecurityImageService provides operations for managing security images.
+service SecurityImageService {
```

Comment:
This way we will have Service per operation. Let's keep it simple by having one service for all operations.